### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by DanNBullock**
Local updates from the Bloomington Pestilli lab as of 8/22/2018

### CODE REVIEW

**Review by francopestilli — COMMENTED** [review]
What is the function: 
display/mbaDisplayConnectome_works.m

Is it used? Shall we delete?
@DanNBullock

File: display/mbaDisplayBrainSlice.m [hunk 1]
```diff
@@ -67,9 +67,11 @@
     h = surf([x.min x.max],[y.min y.max],repmat(z.min, [2 2]),...
         image_rgb,'facecolor','texture','FaceAlpha',alpha);
 end
+axis off
 
 % Rescale the axis to fit the img
 if rescale == 1
+    keyboard
```
Comment (soichih) [inline display/mbaDisplayBrainSlice.m]: I think this should be removed?

Comment (francopestilli) [reply display/mbaDisplayBrainSlice.m]: @DanNBullock line 74 seems like a bug to me also.

Comment (DanNBullock) [reply display/mbaDisplayBrainSlice.m]: If we are being completely honest, I did not make the vast majority of these changes.  I don't actually know the provenance of them.  I'm fine with whatever changes need to be made (i.e. getting rid of this debug stuff)

**Comment by soichih** [comment]
@DanNBullock So there are some code changes that you didn't make but you are making it part of your PR? Are you committing this from some git repo that's communally edited?  

I did noticed that we are still using /N/dc2/projects/lifebid/code/mba on some of our App - probably used by other scripts as well? I can't change how people do things, but for Brainlife App, I think we should be switching to use *this* repo - which is cloned under /N/u/brlife/git/mba.. Do you know if I need to merge some changes made on /N/dc2/projects/lifebid/code/mba to this repo before I switch over?

File: display/mbaDisplayBrainSlice.m (hunk 1 above)
Comment (francopestilli) [reply display/mbaDisplayBrainSlice.m]: @DanNBullock this is the PR stuck

**Comment by soichih** [comment]
Closing this as we don't understand what these changes are. Please send a new PR if necessary.

**Pull request closed without merging**